qemu/tests/balloon_service.py

```diff
@@ -91,7 +91,7 @@ def memory_check(vm, get_polling_output, keyname):
 
         error_context.context("Compare memory from guest with qmp",
                               logging.info)
-        if (abs(guest_mem - stat_memory_qmp)) > (guest_mem * check_mem_ratio):
+        if abs(float(guest_mem - stat_memory_qmp)) / guest_mem > check_mem_ratio:
```

Comment:
the methord is almost the same, so if issue is fixed because need to sleep for a while?